src/app/vault/vault.component.ts

```diff
@@ -246,12 +254,14 @@ export class VaultComponent implements OnInit, OnDestroy {
                 if (cipher.login.username != null) {
                     menu.append(new remote.MenuItem({
                         label: this.i18nService.t('copyUsername'),
+                        accelerator: 'CmdOrCtrl+shift+U',
```

Comment:
Is a shortcut necessary on this menu too? Not sure if it does anything here.

Reply:
I really just put it here so it appears when you right click on the context menu. Just as information. If you feel like it I can remove it.